### PR TITLE
Create LDAP-registry.adoc

### PR DESCRIPTION
**Pull request by dmuelle**

### CODE REVIEW

File: ref/general/LDAP-registry.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+LDAP is an open protocol that supports authentication and authorization services. LDAP specifies the language that application servers use to communicate with the user registries that store security information such as usernames, passwords, and user groups. Although a link:/docs/ref/general/#basic-registry.html[basic user registry] might be sufficient during development and testing of an application, an LDAP user registry provides the extra security that is required for applications in production.
+
+You can configure Open Liberty to retrieve security information from an LDAP server if you are building an application that uses a preexisting user registry. LDAP is the industry standard for directory servers, providing a means to create, modify, search, and delete users and groups in your applications. There are many LDAP servers available, such as the link:https://www.ibm.com/support/knowledgecenter/SSVJJU_6.4.0/com.ibm.IBMDS.doc_6.4/c_po_SDS_overview.html[IBM Security Directory Server] and Microsoft's Active Directory. There are also open source LDAP servers, such as link:http://www.openldap.org/[OpenLDAP] and link:http://directory.apache.org/apacheds/[ApaheDS].
```
Comment (Charlotte-Holt) [inline ref/general/LDAP-registry.adoc]: `ApaheDS`
To
`ApacheDS`

File: ref/general/LDAP-registry.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      baseDN="o=ibm,c=us"
+      ldapType="IBM Tivoli Directory Server" searchTimeout="8m">
+</ldapRegistry>
+----
+For a full list of configuration elements and attributes, see the link:/docs/ref/config/#ldapRegistry.html[LDAP User Registry feature].
+
+== Federated LDAP registries
+You can also federate LDAP user registries so that multiple registries can be referenced together from a local view. You can federate LDAP registries with one another and with custom and basic user registries. For more information, see Federating user registries (link pending).
+
+== Using the Social Media Login feature with LDAP registries.
```
Comment (Charlotte-Holt) [inline ref/general/LDAP-registry.adoc]: Remove the period after `LDAP registries` in the heading

File: ref/general/LDAP-registry.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+You can use Open Liberty's Social Media Login feature to help manage authentication with a configured LDAP registry. Depending how you configure the feature, users can either supply their social media credentials to authenticate to the LDAP registry or choose between their LDAP sign in and social media credentials for authentication. For more information, see the example configurations for the link:/docs/ref/feature/#socialLogin-1.0.html[Social Media Login feature]
```
Comment (Charlotte-Holt) [inline ref/general/LDAP-registry.adoc]: Add a period at the end of the last sentence

**Comment by dmuelle** [comment]
@Charlotte-Holt fixed- thanks

**Comment by dmuelle** [comment]
had to add a section

**Pull request merged**